docs/topics/vertx-dev-guide-additional-resources.adoc

```diff
@@ -2,5 +2,10 @@
 = Additional Resources
 
 * link:http://vertx.io[{VertX} project]
-* link:https://developers.redhat.com/promotions/building-reactive-microservices-in-java/[Building Reactive Microservices in Java book]
-* link:http://escoffier.me/vertx-hol/#_vert_x[Vert.x - From zero to (micro)-hero]
\ No newline at end of file
+* link:http://middlewareblog.redhat.com/2017/05/04/vert-x-for-reactive-programming-in-red-hat-openshift-application-runtimes/[{VertX} for Reactive Programming in Red Hat OpenShift Application Runtimes]
+* link:https://developers.redhat.com/promotions/building-reactive-microservices-in-java/[Building Reactive Microservices in Java]
+* link:https://developers.redhat.com/promotions/vertx-cheatsheet/[{VertX} Cheat Sheet for Developers]
+* link:http://escoffier.me/vertx-hol/#_vert_x[{VertX} - From zero to (micro)-hero]
+* link:https://github.com/cescoffier/rhsummit17-reactive-programming-with-vertx/blob/master/reactive-programming-with-vertx.pdf[Red Hat Summit 2017 Talk - Reactive Programming with {VertX}]
+* link:https://www.redhat.com/en/about/videos/reactive-systems-eclipse-vertx-and-red-hat-openshiftp[Red Hat Summit 2017 Breakout Session - Reactive Systems with {VertX} and Red Hat OpenShift]
```

Comment:
This link does not work, I think there's an extra 'p' at the end of the URL
Should be 
`https://www.redhat.com/en/about/videos/reactive-systems-eclipse-vertx-and-red-hat-openshift`